### ISSUE TITLE
SyncTeX stays busy since 4.8.0 update

### ISSUE THREAD

**Issue by Ayio13** [texstudio-org/texstudio#3655]
### Environment
- TeXstudio: 4.8.0
- Qt: 6.7.0
- OS: Windows 10
- TeX distribution: miktex

### Expected behavior

Synctex file switches from busy to gz, hence allowing jumping from pdf to source inside TeXstudio.

### Actual behavior

Synctex file stays busy, error message is "SyncTeX: Can't rename file.synctex(busy) to file.synctex.gz"
The option of jumping to source doesn't appear in the right click menu.

I've read online that this can occur upon error in tex code or bad file name. However I never had this error before updating, miktex is up to date, it occurs on all tex files, with either pdflatex or lualatex, and the file name doesn't contain special characters. Since the exact same files don't have this issue on an older TeXstudio version (4.7.3), I suspect that it is caused by the update.

### How to reproduce

Compiling generally yields the error. On rare instances syncTeX worked properly, but it never lasted more than 2 compilations.

**Comment by sunderme** [comment]
as a starter: where do you see an error message ?

**Comment by Ayio13** [comment]
Thanks for replying; the error is displayed in the "Messages" window below the editor (not in "log" window).

Upon more testing, I found out that the error occurs only for files that are on an external drive. I also tested with TeXmaker and a similar problem occurs: I can't jump from pdf to source. The main difference is that on TeXmaker the synctex file properly switches from busy to normal. TeXmaker also works fine on local files (not on drive).

So it seems like the main problem comes from my pc, yet I am curious to why both editors don't have the same error (even if it leads to the same practical issue). I'm unsure that this still classifies as a TeXstudio issue.
